tests/virt/cluster/migration_and_maintenance/rbac_hardening/test_migration_rights.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from kubernetes.dynamic.exceptions import ForbiddenError
+from ocp_resources.cluster_role import ClusterRole
+from ocp_resources.role_binding import RoleBinding
+
+from utilities.constants import UNPRIVILEGED_USER
+from utilities.virt import VirtualMachineForTests, fedora_vm_body, migrate_vm_and_verify, running_vm
+
+pytestmark = pytest.mark.rwx_default_storage
+
+
+@pytest.fixture(scope="session")
+def kubevirt_migrate_cluster_role(admin_client):
+    return ClusterRole(name="kubevirt.io:migrate", client=admin_client)
+
+
+@pytest.fixture()
+def unprivileged_client_migrate_rolebinding(admin_client, namespace, kubevirt_migrate_cluster_role):
```

Comment:
the name is misleading as the fixture is using `admin_client`
`unprivileged_user_migrate_rolebinding` would solve this